k9mail/src/main/java/com/fsck/k9/controller/MessagingController.java

```diff
@@ -4753,6 +4753,11 @@ private LocalMessage findNewestMessageForNotificationLocked(Context context, Not
      */
     private void notifyAccount(Context context, Account account,
             LocalMessage message, int previousUnreadMessageCount) {
+    	// if it's quiet time and notifications are disabled, then we shouldn't show a notification
+        if (K9.isQuietTime() && K9.getQuietTimeNotificationEnabled()) {
```

Comment:
You want to return if it's quiet time and notifications during quiet time are **not** enabled. So this is either a logic error or it's bad naming of the setting/field/method.